src/app/shared/PersonName.svelte

```diff
@@ -60,11 +60,10 @@
           </div>
         {/if}
       </div>
-      <div class="flex flex-row items-center gap-1 text-xs">
-        <PersonHandle class="whitespace-nowrap text-accent hidden xs:block" {pubkey}>
-          <span class="opacity-50 hidden xs:block">-</span>
-        </PersonHandle>
-        <p class="whitespace-nowrap opacity-50">{npubDisplay}</p>
+      <div class="flex flex-row items-center gap-2">
+        <PersonHandle class="whitespace-nowrap text-xs text-accent" {pubkey} />
+        <span class="text-xs opacity-75">•</span>
+        <small class="whitespace-nowrap">{npubDisplay}</small>
```

Comment:
I made this change because if there's no handle the bullet is just floating out in space, you'll want to adopt my version here.

Reply:
messed up the merge, probably a good argument in favor of rebase here